mxnet/buildspec-sagemaker.yml

```diff
@@ -0,0 +1,97 @@
+account_id: &ACCOUNT_ID <set-$ACCOUNT_ID-in-environment>
+region: &REGION <set-$REGION-in-environment>
+framework: &FRAMEWORK mxnet
+version: &VERSION 1.9.0
+short_version: &SHORT_VERSION 1.9
+arch_type: x86
+
+repository_info:
+  training_repository: &TRAINING_REPOSITORY
+    image_type: &TRAINING_IMAGE_TYPE training
+    root: !join [ *FRAMEWORK, "/", *TRAINING_IMAGE_TYPE ]
+    repository_name: &REPOSITORY_NAME !join [pr, "-", *FRAMEWORK, "-", *TRAINING_IMAGE_TYPE]
+    repository: &REPOSITORY !join [ *ACCOUNT_ID, .dkr.ecr., *REGION, .amazonaws.com/, *REPOSITORY_NAME ]
+
+  inference_repository: &INFERENCE_REPOSITORY
+    image_type: &INFERENCE_IMAGE_TYPE inference
+    root: !join [ *FRAMEWORK, "/", *INFERENCE_IMAGE_TYPE ]
+    repository_name: &REPOSITORY_NAME !join [pr, "-", *FRAMEWORK, "-", *INFERENCE_IMAGE_TYPE]
+    repository: &REPOSITORY !join [ *ACCOUNT_ID, .dkr.ecr., *REGION, .amazonaws.com/, *REPOSITORY_NAME ]
+
+context:
+  training_context: &TRAINING_CONTEXT
+    entrypoint:
+      source: docker/artifacts/dockerd-entrypoint.py
+      target: dockerd-entrypoint.py
+    deep_learning_container:
+      source: ../../src/deep_learning_container.py
+      target: deep_learning_container.py
+  inference_context: &INFERENCE_CONTEXT
+    config:
+      source: docker/artifacts/config.properties
+      target: config.properties
+    entrypoint:
+      source: docker/artifacts/mms-entrypoint.py
+      target: mms-entrypoint.py
+    ec2-entrypoint:
+      source: docker/artifacts/mms-entrypoint.py
+      target: mms-ec2-entrypoint.py
+    deep_learning_container:
+      source: ../../src/deep_learning_container.py
+      target: deep_learning_container.py
+
+images:
+  BuildSMMXNetCPUTrainPy3DockerImage:
```

Comment:
Naming convention seems to be mismatch between PT and TF dockerfiles

Reply:
I just copy-pasted these from the existing buildspec. Will change these to match the same convention.